src/components/__tests__/ProjectDiscipline-test.js

```diff
@@ -9,6 +9,8 @@ const projectActions = {
   fetchProjects: () => {}
 }
 
+jest.mock('Alert', () => 'Alert')
```

Comment:
This seems odd. Why would this be the only needed change in this test?

Reply:
Not quite sure why either. I updated `jest` as well in the upgrade and I'm guessing it got a little stricter in how it renders things. It shouldn't be a big that we stub this out though because it's a react native component